packages/amplify-graphql-schema-generator/src/generate.ts

```diff
@@ -0,0 +1,53 @@
+import * as os from 'os';
+import { ImportedRDSType } from '@aws-amplify/graphql-transformer-core';
+import { DocumentNode } from 'graphql';
+import { Schema, Engine } from './schema-representation';
+import { generateGraphQLSchema } from './schema-generator';
+import { constructRDSGlobalAmplifyInput } from './input';
+import { MySQLStringDataSourceAdapter } from './datasource-adapter';
+
+const buildSchemaFromString = (stringSchema: string, engineType: ImportedRDSType): Schema => {
+  let schema;
+  let adapter;
+  switch (engineType) {
+    case ImportedRDSType.MYSQL:
+      adapter = new MySQLStringDataSourceAdapter(stringSchema);
+      schema = new Schema(new Engine('MySQL'));
+      break;
+    case ImportedRDSType.POSTGRESQL:
+      // todo change to postgres
+      adapter = new MySQLStringDataSourceAdapter(stringSchema);
+      schema = new Schema(new Engine('Postgres'));
+      break;
+    default:
+      throw new Error('Only MySQL and Postgres Data Sources are supported');
```

Comment:
As above, need to throw from here to prevent execution from proceeding past this error